docs/DeveloperGuide.md

```diff
@@ -735,6 +735,83 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 ## **Appendix: Instructions for manual testing**
 
+Given below are instructions to test the app manually.
 
-### Launch and shutdown
+<div markdown="span" class="alert alert-info">:information_source: **Note:** These instructions only provide a starting point for testers to work on;
+testers are expected to do more *exploratory* testing.
 
+</div>
+
+### Launch and shutdown
+1. Initial launch
+    1. Download the latest release jar file from [here](https://github.com/AY2021S2-CS2103-W16-2/tp/releases).
+    2. Copy the downloaded jar file into an empty folder.
+    3. Nagviate your command prompt to the folder containing the downloaded jar file and run the command `java -jar <fileName>`
+       . Lunach the application by double-clicking only as a last resort if the command does not work.
+
+2. Saving window size and location preferences
+    1. Resize the window to an optimum size. Move the window to a different location. Close the window.
+    2. Re-launch the application by following either the command or by double-clicking.
+        Expected: The most recent window size and location is retained.
+
+3. Saving application launch display list
+    1. launch the application and run either `listorders` , `listcustomers` or `listcheeses`. Close the window.
+    2. Re-launch the application by following either the command or by double-clicking.
+        Expected: The most recent list depending on last `list` command will be displayed.
+
+### Adding a customer
+1. Adding a customer into CHIM
+    1. Prerequisites: Arguments are valid and compulsory parameters are provided. No duplicated phone number exist in CHIM.
+    2. Test case: `addcustomer n/Tim p/96284715 e/tim@example.com a/Blk 55 Woodlands Street 99, #99-23`
+        Expected: item is added into CHIM. If displayed list is customer list , the new customer appears at the bottom.
+            Otherwise, run command `listcustomers`. Details of the new customer is shown in the status message.
```

Comment:
Maybe can rephrase since adding a customer always displays the customer list? Similar for deleting as well

Reply:
sure. Will update in next commit